src/main/java/com/powsybl/openloadflow/sa/extensions/ContingencyLoadFlowParametersJsonSerializer.java

```diff
@@ -0,0 +1,60 @@
+package com.powsybl.openloadflow.sa.extensions;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.google.auto.service.AutoService;
+import com.powsybl.commons.extensions.ExtensionJsonSerializer;
+import com.powsybl.commons.json.JsonUtil;
+import com.powsybl.contingency.Contingency;
+
+import java.io.IOException;
+
+@AutoService(ExtensionJsonSerializer.class)
+public class ContingencyLoadFlowParametersJsonSerializer implements ExtensionJsonSerializer<Contingency, ContingencyLoadFlowParameters> {
+
+    @Override
+    public String getExtensionName() {
+        return "contingency-parameters";
```

Comment:
Adapt it to new class name "contingency-load-flow-parameters"

Reply:
fixed in [74c7960](https://github.com/powsybl/powsybl-open-loadflow/pull/1139/commits/74c79605b4411a6e1ddb50aa9f2a6476625fc91a)